docs/source/docs/quick-start/quick-configure.md

```diff
@@ -0,0 +1,53 @@
+# Quick Configure
+
+## Settings to configure
+
+### Team number
+
+In order for photonvision to connect to the roborio it needs to know your team number.
+
+### Camera Names
+
+You **must** rename your cameras to ensure that every camera has a unique name. This is how we will identify cameras in robot code.
+
+## Pipeline Settings
+
+### Apriltag
+
+When using an Orange pi 5 with an arducam OV9281 teams will usually change the following settings.
```

Comment:
not sure how i feel about this section but spelling consistency, and we should link back to where we explain what these knobs actually do, I don't like just telling people to just go set these magic numbers and they will work